packages/lesswrong/components/sunshineDashboard/ModeratorUserInfo/ContentSummaryRows.tsx

```diff
@@ -54,6 +59,15 @@ export const ContentSummaryRows = ({classes, comments, posts, user, loading}: {
   const hiddenPostCount = user.maxPostCount - user.postCount
   const hiddenCommentCount = user.maxCommentCount - user.commentCount
 
+  const averagePostKarma = posts?.length ? 
+    <LWTooltip title="average karma"><span className={classes.average}>{(posts.reduce((sum, post) => post.baseScore -  + sum, 0) / posts.length ).toFixed(1)}</span></LWTooltip>
```

Comment:
I would pull the calculation out above into in a const so this is easier to read